tests/next_tests/integration_tests/cases.py

```diff
@@ -68,6 +68,7 @@
 IJKField: TypeAlias = gtx.Field[[IDim, JDim, KDim], np.int32]  # type: ignore [valid-type]
 IJKFloatField: TypeAlias = gtx.Field[[IDim, JDim, KDim], np.float64]  # type: ignore [valid-type]
 VField: TypeAlias = gtx.Field[[Vertex], np.int32]  # type: ignore [valid-type]
+VKField: TypeAlias = gtx.Field[[Vertex, KDim], np.int32]  # type: ignore [valid-type]
```

Comment:
Probably would be better to use IField and IKField in the test to avoid adding more types that are not really needed...